client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -294,8 +364,38 @@ export default React.createClass( {
 							</Animate>
 						</button>
 					</div>
-					: null }
+				}
+			</div>
+		);
+	}
+
+	render() {
+		const { site, siteIsJetpack } = this.props;
+
+		return (
+			<div className="site-indicator__wrapper">
+				{ siteIsJetpack && <QuerySiteConnectionStatus siteId={ site.ID } /> }
+				{ siteIsJetpack && <QuerySiteUpdates siteId={ site.ID } /> }
```

Comment:
I don't think we query for site updates any more since they should be part of /me/sites endpoint.